assets/js/week-tracker.js

```diff
@@ -0,0 +1,64 @@
+import Vue from 'vue';
+import axios from 'axios';
+import moment from 'moment';
+
+const app = new Vue({
+  el: '#trackerContainer',
+  data: {
+    weeklyHours: {}
+  },
+
+  created() {
+    this.getWeekly();
+  },
+
+  computed: {
+    displayedWeek() {
+      const urlDate = window.location.search.split('=')[1];
```

Comment:
This seems a bit brittle. Can we parse for the specific param instead, if we end up with another query param in the url at any point this will fall over